dds/DCPS/SporadicTask.h

```diff
@@ -69,17 +72,37 @@ class SporadicTask : public RcEventHandler {
 private:
   WeakRcHandle<ReactorInterceptor> interceptor_;
   bool scheduled_;
+  enum {
+    NONE,
+    SCHEDULE,
+    CANCEL
```

Comment:
Use enumerator names that are less likely to have naming collisions with other identifiers including macros.